onnx2kerastl/pooling_layers.py

```diff
@@ -68,7 +71,18 @@ def convert_maxpool(node, params, layers, lambda_func, node_name, keras_name):
             name=keras_name,
             data_format='channels_first'
         )
-
+    ceil_mode = params['ceil_mode'] if 'ceil_mode' in params else False
+    if ceil_mode:
+        if pad == 'valid':
+            output_shape = ((np.array(input_0.shape[-len(kernel_shape):]) - np.array(kernel_shape)) / np.array(
+                stride_shape)) + 1
+        else:
+            output_shape = np.floor((np.array(input_0.shape[-len(kernel_shape):]) - 1) / np.array(stride_shape)) + 1
+        if not output_shape[0].is_integer() or not output_shape[1].is_integer():
+            padding = [0 if output_shape[i].is_integer() else stride_shape[i] for i in range(len(kernel_shape))]
+            rand_string = ''.join(random.choices(string.ascii_uppercase + string.digits, k=3))
+            layers[node_name + "_pre_" + rand_string] = keras.layers.ZeroPadding2D(((0,padding[0]), (0,padding[1])))(input_0)
+            input_0 = layers[node_name + "_pre_" + rand_string]
```

Comment:
Nice!!

Reply:
True - added this case more completely and a test that tests it as a unit test